python/GangaDirac/Lib/Splitters/OfflineGangaDiracSplitter.py

```diff
@@ -14,10 +19,21 @@
 LFN_parallel_limit = 250
 limit_divide_one = 1. / float(LFN_parallel_limit)
 
+def wrapped_execute(command, expected_type):
+    """
+    A wrapper around execute to protect us from commands which had errors
+    """
+    try:
+        result = execute(command)
+        assert isinstance(result, expected_type)
+    except AssertionError:
+        raise SplittingError("Output from DIRAC expected to be of type: '%s', we got the following: '%s'" % (expected_type, result))
```

Comment:
Why `assert` then catch `AssertionError`? Why not just raise `SplittingError` directly?
```python
result = execute(command)
if not isinstance(result, expected_type):
    raise SplittingError("Output from DIRAC expected to be of type: '%s', we got the following: '%s'" % (expected_type, result))
return result
```